.yetus/blanks-tabs.txt

```diff
@@ -3,3 +3,4 @@
 .*\.dts
 .*\.md
 ^.codespellignorelines
+pkg/dom0-ztools/rootfs/usr/bin/rungetty.sh
```

Comment:
why do you want to ignore blank tabs?

Reply:
As far as I understand, because it's not our code originally )